### PR TITLE
Adds new deathsquad name and two leader titles

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
"I'm told my ops are hidden as an HR expense in the TranStar books. It's true, I do solve people problems."
Thread got me thinking about deathsquad and I came up with this
(Walther) Dahl belongs to Prey (2017), which was developed by Arkane Studios and published by Bethesda Studios, I'm putting this here hoping lawyers won't gut me
Also adds a "leader name" system to strike teams but currently only used by deathsquad
:cl:
 * tweak: Added new deathsquad commander name and titles.

### CODE REVIEW

File: code/game/striketeams/nanotrasen_deathsquad.dm [hunk 1]
```diff
@@ -12,15 +12,16 @@
 
 /datum/striketeam/deathsquad/create_commando(obj/spawn_location, leader_selected = 0)
 	var/mob/living/carbon/human/new_commando = new(spawn_location.loc)
-	var/commando_leader_rank = "Major"
+	var/commando_leader_rank = pick("Major", "Rescue Team Leader", "Commander")
 	var/commando_rank = pick("Corporal", "Sergeant", "Staff Sergeant", "Sergeant 1st Class", "Master Sergeant", "Sergeant Major")
 	var/commando_name = pick(last_names)
+	var/commando_leader_name = pick("Creed", "Walther Dahl")
```
Comment (MadmanMartian) [inline code/game/striketeams/nanotrasen_deathsquad.dm]: Aren't these usually last names?

Commander Dahl's got a better ring to it than Commander Walther Dahl

File: code/game/striketeams/nanotrasen_deathsquad.dm [hunk 2]
```diff
@@ -12,15 +12,16 @@
 
 /datum/striketeam/deathsquad/create_commando(obj/spawn_location, leader_selected = 0)
 	var/mob/living/carbon/human/new_commando = new(spawn_location.loc)
-	var/commando_leader_rank = "Major"
+	var/commando_leader_rank = pick("Major", "Rescue Team Leader", "Commander")
```
Comment (SysadminJeroen) [inline code/game/striketeams/nanotrasen_deathsquad.dm]: This is a Death Squad, why have one of the ranks be `Rescue Team Leader` if they're going to assblast everyone?

File: code/game/striketeams/nanotrasen_deathsquad.dm (hunk 1 above)
Comment (boy2mantwicethefam) [reply code/game/striketeams/nanotrasen_deathsquad.dm]: I considered that

File: code/game/striketeams/nanotrasen_deathsquad.dm (hunk 2 above)
Comment (Kurfursten) [inline code/game/striketeams/nanotrasen_deathsquad.dm]: Rescue Team Leader is pretty long for a title. 26 character is MAX_NAME_LEN and this takes up 18 letters.

Also seems a bit off for a deathsquad title.

**Review by SysadminJeroen — CHANGES_REQUESTED** [review]

**Comment by kilozombie** [comment]
>(Walther) Dahl belongs to Prey (2017), which was developed by Arkane Studios and published by Bethesda Studios, I'm putting this here hoping lawyers won't gut me

image

File: code/game/striketeams/nanotrasen_deathsquad.dm (hunk 2 above)
Comment (boy2mantwicethefam) [reply code/game/striketeams/nanotrasen_deathsquad.dm]: It's intentionally misleading

Comment (boy2mantwicethefam) [reply code/game/striketeams/nanotrasen_deathsquad.dm]: Not sure what to shorten it to but it's intentionally misleading

**Comment by Blithering** [comment]
Alternate deathsquad titles:
Janitorial Division
Asset Management
Nanotrasen Pizza Delivery
HR (Liquidations)

Will keep adding if I think of more good ones.

File: code/game/striketeams/nanotrasen_deathsquad.dm (hunk 2 above)
Comment (MadmanMartian) [reply code/game/striketeams/nanotrasen_deathsquad.dm]: Rescue leader?

Comment (boy2mantwicethefam) [reply code/game/striketeams/nanotrasen_deathsquad.dm]: Sure

**Pull request merged**